### ISSUE TITLE
Password is not accepted

### ISSUE THREAD

**Issue by drbugfinder** [moosethegoose2213/automator-for-barrykn-micropatcher#17]
I downloaded the current version (2.0) and in the "Download Installer App" step, the password entry field does not accept a password. The entry field just re-appears every time I hit "authorize".

**Comment by moosethegoose2213** [comment]
If this is a blank password, this is known. There isn't any plan to fix this, as having no password is a big security risk that we do not support the use of. Is it a blank password, or does it have any special characters?

**Comment by drbugfinder** [comment]
It's not an empty password, but it contains a Dollar symbol

**Comment by drbugfinder-work** [comment]
I created a new user with a very simple password without special characters and it worked without any problems. So there seems to be a problem with the password string escape.

**Comment by moosethegoose2213** [comment]
Yeah, sorry about not getting back to you about that! That is a good workaround, and thank you for reporting the bug. Sorry for not suggesting this before!
